mula/scheduler/server/handlers/schedules.py

```diff
@@ -246,3 +254,20 @@ def patch(self, schedule_id: uuid.UUID, schedule: serializers.SchedulePatch) ->
             ) from exc
 
         return updated_schedule
+
+    def delete(self, schedule_id: uuid.UUID) -> None:
+        try:
+            self.ctx.datastores.schedule_store.delete_schedule(schedule_id)
+        except storage.errors.StorageError as exc:
+            raise fastapi.HTTPException(
+                status_code=fastapi.status.HTTP_500_INTERNAL_SERVER_ERROR,
+                detail=f"error occurred while accessing the database [exception: {exc}]",
+            ) from exc
+        except Exception as exc:
+            self.logger.exception(exc)
```

Comment:
I would suggest using an error message here instead, otherwise it's a bit hard to find what the current error is about.